app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/FolderController.kt

```diff
@@ -137,44 +133,24 @@ class FolderController @Inject constructor(
         private val isRootFolder = "${Folder.parentsPropertyName}.@count == 0"
 
         //region Queries
-        private fun getFoldersQuery(realm: TypedRealm, onlyRoots: Boolean = false): RealmQuery<Folder> {
+        private fun getFoldersQuery(
+            realm: TypedRealm,
+            onlyRoots: Boolean = true,
+            excludeDrafts: Boolean = false,
+        ): RealmQuery<Folder> {
```

Comment:
We never want to get children, because we can't correctly order them via Realm.
There is only 1 exception in the app where we want to get them, this `onlyRoots` is only here for this situation.
I can rename it in `withChildren` if you prefer, and invert the default value.

Reply:
The parameter name ultimately doesn't really botter me that much, whichever you prefer is fine by me. What bothers me more is that calling `folderController.getFoldersAsync()` doesn't return all the folders, contrary to what its name implies. This aspect is the part that's not intuitive for anyone reading this code with no context, in my opinion.

It would be more straightforward If it was always called either as `folderController.getFoldersAsync(onlyRoots = true)`, probably the best solution, or even as `folderController.getRootFoldersAsync()`. Even though I don't exactly advise that last one because when want the children it would be strange to call a `getRootFolders` method to then precise we also want the children

Reply:
Done.